src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -115,14 +116,18 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         if (file_exists($dir = $container->getParameter('kernel.root_dir').'/Resources/views')) {
+            if ($dir !== $defaultTwigPath) {
+                @trigger_error(sprintf('"%s" This templates directory is deprecated since 4.2, use "%s" instead.', $dir, $defaultTwigPath), E_USER_DEPRECATED);
```

Comment:
```diff
- "%s" This templates directory is deprecated since 4.2, use "%s" instead.
+ Templates directory "%s" is deprecated since 4.2, use "%s" instead.
```

? (same below)

Reply:
Hey! fixed, thanks.